msteams-platform/tabs/how-to/personal-tab-navigation.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Personal tab app navigation
+description: Learn about the options for navigation within a personal tab app using the TeamsJS library.
+ms.date: 10/26/2022
+ms.topic: conceptual
+ms.custom: m365apps
+ms.localizationpriority: medium
+---
+
+# Navigate within a personal tab app
+
+The TeamsJS library provides support for navigation within a personal app. This article discusses the options available, which includes types of navigation such as that between tabs within the app or through use of Teams UI components like the back button.
+
+[!INCLUDE [sdk-include](~/includes/sdk-include.md)]
+
+Personal tabs provide a great way to enhance the Microsoft Teams experience. Using personal tabs, you can provide a user access to their application right within Teams, without the user having to leave the experience or sign in again. For more information about personal tabs and how you can extend them across Microsoft 365 products, see [Build tabs for Teams](~/tabs/what-are-tabs.md) and [Extend a Teams personal tab across Microsoft 365](~/m365-apps/extend-m365-teams-personal-tab.md).
```

Comment:
This reads a little ambiguous to me (seems more written towards admins rather than developers): *Using personal tabs, you can provide a user access to their application right within Teams* 
. . . how about: 
With personal tabs, you can provide users access to **your web** application right within Teams

Reply:
This has been updated